packages/common/src/chains/mainnet.json

```diff
@@ -104,6 +104,11 @@
       "block": null,
       "timestamp": "1681338455",
       "forkHash": "0xdce96c2d"
+    },
+    {
+      "name": "cancun",
+      "block": null,
+      "forkHash": null
```

Comment:
I wonder if me might always want to take the sematics here:

If there is an entry on something with `null`, there is (will be) some value, we just don't know (yet) what it is, and completely leaving when there is just not a value.

This would be an argument to also add `"timestamp": null` for these HF entries (while having (obviously) left the `timestamp` entry for a HF like London).

I have absolutely no idea/overview, if this has practical implications or not, but it might help that we can do some things slightly cleaner already or in the future.

Totally no strong opinion here though, might help a bit, might be a total "non-thing", feel free to add or leave (if you e.g. do not want to re-run CI).